pkgs/development/python-modules/durus/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "durus";
+  version = "4.2";
+
+  src = fetchFromGitHub {
+    owner = "nascheme";
+    repo = "durus";
+    rev = "v${version}";
+    sha256 = "sha256:1kjiyfv5cb36g563bawj9hx563r3shhmcq77flkqb93b80qvkbqv";
+  };
+
+  nativeBuildInputs = [ ];
+  buildInputs = [  ];
+
+  doCheck = false;
```

Comment:
Why are you not executing the test suite? If it's not possible in the Nix build sandbox or unfeasible, please add a comment noting why.

Test suites are relatively important for python packages as they are virtually the only sanity checking possible with it lacking type checking etc.

If you can't enable tests, add an `pythonImportsCheck` at least.

Reply:
added a comment explaining why tests are ommited.